packages/client/manifest.json

```diff
@@ -5017,6 +5059,20 @@
         "section": true,
         "name": "Buttons",
         "settings": [
+          {
+            "type": "text",
+            "key": "saveButtonLabel",
+            "label": "Save Button Label",
```

Comment:
NAB: We should try to use consistent casing for settings, and the standard is to only capitalise the first letter of the first word, rather than every word. Would also be preferable to shrink this slightly to fit into a single line if possible.